src/Test/SymfonyTest.php

```diff
@@ -108,14 +108,21 @@ protected function getProxy()
      *
      * @return Symfony
      */
-    protected function getProxyClient()
+    protected function getProxyClient(array $options = array())
     {
+        $options = array_merge(
+            [
+                'purge_method' => 'NOTIFY'
```

Comment:
we should only change the method when the test wants to check something with it.